src/Containers/VendorForm/VendorForm.js

```diff
@@ -1,34 +1,78 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { useDropzone } from 'react-dropzone'
```

Comment:
useDropzone interesting.  Guessing that is a react tool for uploading an image.  Looks good